docs/user/installation.rst

```diff
@@ -10,17 +10,22 @@ Express Install
 
 The fastest way to set up dependencies for garage is via running the setup script.
 
+Clone this repo and navigate to its directory.
+
+A MuJoCo key is required for installation.
+
 - On Linux, run the following:
 
 .. code-block:: bash
 
-    ./scripts/setup_linux.sh
+    ./scripts/setup_linux.sh --mjkey path-to-your-mjkey.txt
```

Comment:
You should probably mention the other options, such as the one which amends bashrc for you. I think you would want that for a truly "express" install.

Reply:
Peter, you can get the argument name by running `./scripts/setup_linux.sh -h`. Also, could you please emphasize users to run the script at the root at garage?
I see you wrote the command to be executed at the root of the garage folder, but I have noticed sometimes people run the script inside the `scripts` folder and it won't work that way.